### PR TITLE
[OSDOCS-5095]: Nutanix OoT provider is GA

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.13+

Issue:
[OCPCLOUD-1405](https://issues.redhat.com//browse/OCPCLOUD-1405) | [OSDOCS-5095](https://issues.redhat.com//browse/OSDOCS-5095)

Link to docs preview:
[Cluster Cloud Controller Manager Operator](https://56422--docspreview.netlify.app/openshift-enterprise/latest/operators/operator-reference.html#cluster-cloud-controller-manager-operator_cluster-operators-ref)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@elmiko this is ready for review when you have time, PTAL

File: modules/cluster-cloud-controller-manager-operator.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 [NOTE]
 ====
-This Operator is only fully supported for Microsoft Azure Stack Hub, {rh-openstack-first}, and VMware vSphere.
+This Operator is fully supported for Microsoft Azure Stack Hub, Nutanix, {rh-openstack-first}, and VMware vSphere.
```
Comment (elmiko) [inline modules/cluster-cloud-controller-manager-operator.adoc:13]: when we say fully supported, are we talking about the GA release status?

**Review by elmiko — COMMENTED** [review]
looks good to me, but i have a question

File: modules/cluster-cloud-controller-manager-operator.adoc (hunk 1 above)
Comment (jeana-redhat) [reply modules/cluster-cloud-controller-manager-operator.adoc:13]: Good point, better to just say GA when we mean GA. Thanks!

**Comment by jeana-redhat** [comment]
@huali9 this one is ready too, PTAL :)

**Review by elmiko — COMMENTED** [review]
/lgtm

(we talked about the vsphere ga issue in slack)

**Pull request merged**